packages/polywrap/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "polywrap"
+description = "Polywrap Client implementation in rust"
```

Comment:
Not sure what to put here, but we could use a better description

Reply:
hehe open to suggestions

Reply:
We'll figure it out :D
Let's ship it!